test/sp.config.configLoader_test.js

```diff
@@ -52,9 +51,16 @@ describe('Configuration loader', function () {
     }
   });
 
-  it('should load without configuration', function(done) {
+  it('should load configuration with only api key', function(done) {
```

Comment:
It's unclear to me what this test is asserting (maybe it needs a more descriptive statement).  Other than that I think this PR looks great.